File: src/bumperbot_example/nodes/turtlesim_kinematics_node.cpp
#include "bumperbot_example/turtlesim_kinematics.h"

int main(int argc,char ** argv)
{
    ros::init(argc,argv,"turtlesim_kinematics_node");
    Turtlesim_Kinematics turtle_kinematics;
    ros::spin();
    return 0;
}

File: src/bumperbot_example/nodes/turtlesim_kinematics_node.py
#!/usr/bin/env python3
import rospy
from bumperbot_example.turtlesim_kinematics import TurtlesimKinematics

if __name__== '__main__':
    rospy.init_node('tutlesim_kinematics_node')
    turtlesim_kinematics = TurtlesimKinematics()
    rospy.loginfo("info message")
    rospy.spin()
    
    